bacon/model/StorageCouchBaseDB.swift

```diff
@@ -116,6 +120,7 @@ class StorageCouchBaseDB {
     func clearTransactionDatabase() throws {
```

Comment:
Dayum imagine if this gets accidentally called... GG

Reply:
Hahaha